session/txn.go

```diff
@@ -532,9 +540,10 @@ func (txnFailFuture) Wait() (uint64, error) {
 
 // txnFuture is a promise, which promises to return a txn in future.
 type txnFuture struct {
-	future   oracle.Future
-	store    kv.Storage
-	txnScope string
+	future               oracle.Future
+	store                kv.Storage
+	txnScope             string
+	enableTemporaryFlags bool
```

Comment:
Can you add some comments for the field `enableTemporaryFlags`.

Reply:
Sure. But first we need to decide whether this field is needed.

Reply:
I've removed it since `compensating lock on read` was abandoned.